src/amo/components/CollectionManager/index.js

```diff
@@ -154,7 +155,25 @@ export class CollectionManagerBase extends React.Component<Props, State> {
     event: ElementEvent<HTMLInputElement | HTMLTextAreaElement>
   ) => {
     event.preventDefault();
-    this.setState({ [event.target.name]: event.target.value });
+    const { name, value } = event.target;
+    const { creating } = this.props;
+
+    if (value === null || typeof value === 'undefined') return;
```

Comment:
> We always use (curly) brackets.

lets put a linting rule for this?

Reply:
can we do this?

Reply:
> !value will not work, as it will consider blank spaces too.

ok I was not sure to be honest, 👍

Reply:
> can we do this?

I will give a quick look, meanwhile I will update PR with `{}` so that this can be merged.



> ok I was not sure to be honest, 👍

Me too, I encountered once I wasnt able to enter space in `name`